replica/rest/model.go

```diff
@@ -210,13 +196,16 @@ func NewReplica(context *api.ApiContext, state replica.State, info replica.Info,
 	r.SectorSize = info.SectorSize
 	r.Size = strconv.FormatInt(info.Size, 10)
 	r.RevisionCounter = strconv.FormatInt(info.RevisionCounter, 10)
+	r.UsedBlocks, r.UsedLogicalBlocks = "0", "0" // replica must be initializing
```

Comment:
Let's put NA over here instead of "0". "0" is misleading and shocking.
And let's mention in comment that this info  is still not available.

Reply:
It is converted to int64 later, so putting default value.

Reply:
But make sure when user is fetching this value, he doesn't get 0.